test-tools/issuer-front-end/src/Main.tsx

```diff
@@ -504,12 +509,28 @@ export default function Main(props: WalletConnectionProps) {
                                     onClick={() => {
                                         setTxHash('');
                                         setTransactionError('');
+
+                                        const schemaCredentialURL = document.getElementById(
+                                            'credentialSchemaURL'
+                                        ) as unknown as HTMLFormElement;
+
+                                        const exampleCredentialSchema = {
+                                            schema_ref: {
+                                                hash: {
+                                                    None: [],
+                                                },
+                                                url: EXAMPLE_CREDENTIAL_SCHEMA,
+                                            },
+                                        };
+
                                         const tx = createNewIssuer(
                                             connection,
                                             account,
                                             issuerMetaData,
                                             issuerKeys?.verifyKey || '',
-                                            JSON.stringify(schemaCredential),
+                                            schemaCredentialURL.value === ''
+                                                ? JSON.stringify(exampleCredentialSchema)
+                                                : JSON.stringify(schemaCredential),
```

Comment:
This looks good. Although I'm not sure why JSON.stringify is actually needed there.

Reply:
Removed the `JSON.stringify` and passed the object instead.